transport/src/test/java/me/uport/sdk/transport/ResponseParserTest.kt

```diff
@@ -9,7 +9,11 @@ import org.junit.rules.ExpectedException
 
 class ResponseParserTest {
 
-    private val valid = listOf(
+    @get:Rule
```

Comment:
we should be using `assertk` APIs instead of `ExpectedException` rule since it allows assertions of exceptions for loops as well and the API is a little more expressive

Reply:
Fixed in an earlier branch. This will be resolved when I rebase on `develop`